docs/content/preview/drivers-orms/rust/_index.md

```diff
@@ -16,10 +16,12 @@ showRightNav: true
 
 The following project is recommended for implementing Rust applications using the YugabyteDB YSQL API.
 
-| Project | Documentation and Guides | Latest Driver Version | Supported YugabyteDB Version | Example Apps |
-| ------- | ------------------------ | --------------------- | ---------------------------- | ------------ |
+| Project | Documentation | Latest Driver Version | YugabyteDB Support | Example&nbsp;Apps |
+| ------- | ------------- | --------------------- | ------------------ | ------------ |
 | YugabyteDB Rust-Postgres Smart Driver [Recommended] | [Documentation](yb-rust-postgres/)<br /> [Reference](rust-postgres-reference/) | [yb-postgres](https://crates.io/crates/yb-postgres) (synchronous YSQL client): v0.19.7-yb-1-beta.3 <br/> [yb-tokio-postgres](https://crates.io/crates/yb-tokio-postgres) (asynchronous YSQL client): v0.7.10-yb-1-beta.3 | 2.19 and later | |
-| Diesel | [Documentation](diesel/) <br/> [Hello World](../orms/rust/ysql-diesel/) | | |[Diesel app](https://github.com/YugabyteDB-Samples/orm-examples/tree/master/rust/diesel) |
+| Diesel | [Documentation](diesel/) <br/> [Hello World](../orms/rust/ysql-diesel/) | Use [rust-postgres](https://github.com/sfackler/rust-postgres) | |[Diesel&nbsp;app](https://github.com/YugabyteDB-Samples/orm-examples/tree/master/rust/diesel) |
+
+Note that Diesel is not compatible with the YugabyteDB Rust smart driver; use the upstream rust-postgres driver instead.
```

Comment:
```suggestion
Note that Diesel uses the `pq-sys` crate instead of the rust-postgres driver and since the YugabyteeDB Rust smart driver is based on rust-postgres, diesel does not work with our smart driver.
```